marketplace-ui/src/main.ts

```diff
@@ -3,7 +3,14 @@
 import { bootstrapApplication } from '@angular/platform-browser';
 import { appConfig } from './app/app.config';
 import { AppComponent } from './app/app.component';
+import { SecurityMonitorComponent } from './app/modules/security-monitor/security-monitor.component';
+import { provideHttpClient } from '@angular/common/http';
 
-bootstrapApplication(AppComponent, appConfig).catch(err => {
-  throw err;
-});
+const currentPath = window.location.pathname;
```

Comment:
Why do you not define <app-security-monitor> in <app-root> and  define path in app.routes.ts?